README.md

```diff
@@ -37,7 +38,6 @@ Run this in a terminal:
 cleos create account eosio talk EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV
 cleos set code talk talk.wasm
 cleos set abi talk talk.abi
-
```

Comment:
This extra space is intentional. It simplifies copy-paste.

Reply:
Noted. I'll fix the rest.